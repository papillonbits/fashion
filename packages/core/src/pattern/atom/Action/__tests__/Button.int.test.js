import { regular } from './Button.int.story'

describe('<Button />', () => {
  describe('Render', () => {
    test('must match regular', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
