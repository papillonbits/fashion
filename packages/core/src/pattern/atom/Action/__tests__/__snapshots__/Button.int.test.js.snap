// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> Render must match regular 1`] = `
<div>
  <button
    class="button"
    type="button"
  >
    click me
  </button>
</div>
`;
